### ISSUE TITLE
Code Coverage

### ISSUE THREAD

**Issue by shadowtime2000** [alephjs/aleph.js#168]
In 1.8.0 LCOV got support with code coverage and denos testing thing so I think we should use a code coverage service so we can make sure everything is being tested. We could use coveralls.io or codecov, I prefer coveralls but I have never used codecov.

**Comment by Hunam6** [comment]
From what I've seen, Codecov is more modern, it has a cleaner interface and seems to be better integrated with Github but Coveralls is older so maybe they have better support 🤷. But in my opinion, as Codecov seems more modern, it will surely have support for Deno sooner.

**Comment by shadowtime2000** [comment]
Yeah, I have only used coveralls in Node projects because of it's weird set up necessary with Travis CI, so I think we could look more into codecov.

**Comment by shadowtime2000** [comment]
> But in my opinion, as Codecov seems more modern, it will surely have support for Deno sooner.

@Hunam6 actually you can do it already using their GH action that can be added to workflows

**Comment by shadowtime2000** [comment]
@ije Thoughts on this?

**Comment by ije** [comment]
yeah, let's try it, but need to add more testings firstly...
